backend/src/main/scala/cromwell/backend/model/MemorySize.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.backend.validation
+package cromwell.backend.model
```

Comment:
I'm aware of what MVC is. I was asking why you think we're doing MVC here and what makes this an M